slices/libgdk-pixbuf-2.0-0.yaml

```diff
@@ -0,0 +1,29 @@
+package: libgdk-pixbuf-2.0-0
+
+essential:
+  - libgdk-pixbuf-2.0-0_copyright
+
+slices:
+  bins:
+    essential:
+      - libgdk-pixbuf-2.0-0_libs
+    contents:
+      /usr/lib/*-linux-*/gdk-pixbuf-2.0/gdk-pixbuf-query-loaders:
```

Comment:
can you please add a test for this binary?

Reply:
Thanks for catching that, ported the tests from [24.04](https://github.com/lengau/chisel-releases/blob/1947fcf209051e5eaaeccbe2efd7b7af33386de8/tests/spread/integration/libgdk-pixbuf-2.0-0/task.yaml) in 6fc738e